DiffPlex.ConsoleRunner/Program.cs

```diff
@@ -32,31 +32,57 @@ private static void Main()
                 Console.WriteLine(line.Text);
             }
         }
+        internal const string OldText =
+            @"ABCDEFG hijklmn 01234567 _!# 98?
```

Comment:
@dualbios  Can you undo changes to this file?

Reply:
fixed in 1fa5e0d